packages/smooth_app/lib/pages/user_management/sign_up_page.dart

```diff
@@ -348,6 +353,15 @@ class _SignUpPageState extends State<SignUpPage> with TraceableClientMixin {
     }
     if (status.error != null) {
       await LoadingDialog.error(context: context, title: status.error);
+
+      if (status.status == 400) {
+        if (status.error?.contains('username already exists') == true) {
```

Comment:
will it stand localized responses from the server ?

Reply:
The server side change is not yet implemented, so I don’t know

Reply:
I have opened: https://github.com/openfoodfacts/openfoodfacts-server/issues/7039